libs/core/src/lib/bar/directives/bar-element.directive.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { BarModule } from '../bar.module';
+
+@Component({
+    template: `
+        <div #directiveElement fd-bar-element [fullWidth]="true">Bar Element Test</div>
+    `
+})
+class TestComponent {
+    @ViewChild('directiveElement', { static: false })
+    ref: ElementRef;
+}
+
+describe('ActionBarActionsDirective', () => {
```

Comment:
Incorrect name, applies to other tests in this pr as well

Reply:
Thanks for noticing this. I fixed it.